src/lib/include/libsed.h

```diff
@@ -96,13 +96,32 @@ struct sed_opalv200_supported_feat {
 	uint8_t reserved2[5];
 } __attribute__((__packed__));
 
+struct sed_tper_properties {
+	uint64_t MaxComPacketSize;
+	uint64_t MaxPacketSize;
+	uint64_t MaxIndTokenSize;
+	uint64_t MaxPackets;
+	uint64_t MaxSubpackets;
+	uint64_t MaxMethods;
+	uint64_t MaxResponseComPacketSize;
+	uint64_t MaxSessions;
+	uint64_t MaxAuthentications;
+	uint64_t MaxTransactionLimit;
+	uint64_t DefSessionTimeout;
+	uint64_t MaxSessionTimeout;
+	uint64_t MinSessionTimeout;
+	uint64_t DefTransTimeout;
+	uint64_t MaxTransTimeout;
+} __attribute__((__packed__));
+
 struct sed_opal_level0_discovery {
 	struct sed_tper_supported_feat sed_tper;
 	struct sed_locking_supported_feat sed_locking;
 	struct sed_geometry_supported_feat sed_geo;
 	struct sed_datastr_table_supported_feat sed_datastr;
 	struct sed_opalv100_supported_feat sed_opalv100;
 	struct sed_opalv200_supported_feat sed_opalv200;
+	struct sed_tper_properties sed_tper_props;
```

Comment:
Tper properties doesn't belong here.
According to spec it is Level 1 discovery (see page 67 of TCG Storage Architecture Core Spec)

Reply:
will remove it and add a new structure

Reply:
Done